aurora_module/outputs.tf

```diff
@@ -0,0 +1,49 @@
+/**
+ * Copyright (c) 2011 - 2017, Coveo Solutions Inc.
+ */
+
+output "endpoint" {
+  value = "${aws_rds_cluster.rds_db_cluster.endpoint}"
+}
+
+output "security_group_id" {
+  value = "${aws_security_group.rds_cluster_security_group.id}"
+}
+
+output "master_username" {
+  sensitive = true
+  value     = "${aws_rds_cluster.rds_db_cluster.master_username}"
+}
+
+output "master_password" {
```

Comment:
I don't think we need to output the master password, because the master password was passed as a variable. I don't see the need for it 🙂

Reply:
My bad, this shouldn't be there. :)